MessageBird/Client.cs

```diff
@@ -79,6 +79,20 @@ public Message ViewMessage(string id)
             return result.Object as Message;
         }
 
+        public MessageList ListMessages(string status = "", int limit = 20, int offset = 0)
+        {
+            var messageLists = new MessageLists();
+
+            var messageList = (MessageList)messageLists.Object;
+            messageList.Limit = limit;
+            messageList.Offset = offset;
+            messageList.Status = status;
```

Comment:
is it possible to do this using the constructor instead, like for example:
`var resource = new CallLists(new CallList { Limit = limit, Page = page });`
just because this is the way we do it for all the other functions and it would keep the code consistent